crates/prompt_gateway/tests/integration.rs

```diff
@@ -332,7 +333,7 @@ llm_providers:
 
 overrides:
   # confidence threshold for prompt target intent matching
-  prompt_target_intent_matching_threshold: 0.6
+  prompt_target_intent_matching_threshold: 0.0
```

Comment:
why we changing this?

Reply:
this is for testing, I changed nan  -> 0.0 when embeddings is not found. So had to lower the threshold so we go to weather endpoint